content/content-strategy/define-user-content-needs/create-actionable-artefacts/journey-maps/cards.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: cards/cards
+cards:
+  - headline: User research
+    text: Do user research in every iteration of development 
+    link: https://guides.service.gov.au/user-research/
```

Comment:
Use the relative path instead of the whole URL

Reply:
Steve fixed